README.md

```diff
@@ -129,31 +129,39 @@ Verify that package name is selected. Just save and exit.
 
 ## Customising your Linux Kernel
 
-Kernel configuration is saved at target/linux/pistachio/config-4.1
-e.g. This the one for the IMG pistachio board as it has some useful defaults.
+Kernel configuration is saved at target/linux/pistachio/config-4.4. This is the one for the IMG pistachio board as it has some useful defaults.
 
 To customise to suit your requirements use the following command:
 
     $ make kernel_menuconfig
 
-If you change any option you need and then save & quit GUI, the changed configuration will get written into target/linux/pistachio/config-4.1
+If you change any option you need and then save & quit GUI, the changed configuration will get written into target/linux/pistachio/config-4.4
 
 For more details please refer to [OpenWrt Build System]("http://wiki.openwrt.org/doc/howto/build")
 
 ## Adding Linux Kernel patches
+
 Linux kernel patches are added at :-
 
     target/linux/pistachio/patches-<kernel_version>/
 
 * All kernel patches are created from the linux kernel hosted [here](https://github.com/CreatorDev/linux).
 * Kernel patches for specific kernel version used in OpenWrt are prepared from corresponding branches in [linux](https://github.com/CreatorDev/linux) repo. e.g.
 
-        target/linux/pistachio/patches-4.1/
+        target/linux/pistachio/patches-4.4/
 
-contains patches created from [openwrt-4.1.13](https://github.com/CreatorDev/linux/tree/openwrt-4.1.13) branch.
+contains patches created from [openwrt-4.4.14](https://github.com/CreatorDev/linux/tree/openwrt-4.4.14) branch.
 * For adding the kernel patch in OpenWrt, create a PR in [linux](https://github.com/CreatorDev/linux) with the change, and also create a PR with the patch of the change in OpenWrt repository.
 * Following command can be used for creating the kernel patch :-
 
         git format-patch <commit_id> --keep-subject --start-number <number>
 
 *NOTE :* Number should the next one from the last patch already added in OpenWrt.
+
+----
+
+## Contributing
+
+We welcome all contributions to this project and we give credit where it's due. Anything from enhancing functionality to improving documentation and bug reporting - it's all good.
+
+For more details about the Contributor's guidelines, refer to the [contributor guide](https://github.com/CreatorKit/creator-docs/blob/master/ContributorGuide.md).
```

Comment:
@abhijit-mahajani-imgtec yes definitely.  But, I guess the first step would be to have the contributor guide on docs website, then a separate PR can be added to change the link to point to that location